librz/util/str_search.c

```diff
@@ -0,0 +1,280 @@
+#include <rz_util/rz_str_search.h>
+#include <rz_util/rz_utf8.h>
+#include <rz_util/rz_utf16.h>
+#include <rz_util/rz_utf32.h>
+
+static RzUtilStrScanOptions g_opt = {
+	.buf_size = 2048,
+	.max_uni_blocks = 4,
+	.min_str_length = 4
+};
+
+RZ_API void rz_detected_string_free(RzDetectedString *str) {
+	if (str) {
+		free(str->string);
+		free(str);
+	}
+}
+
+RZ_API int rz_scan_strings(RzList *list, RzBuffer *buf_to_scan,
+	const ut64 from, const ut64 to, int min_str_length, RzStrEnc type) {
+
+	// list cannot be NULL
```

Comment:
No need for these comments in both cases.

Reply:
Removed